russstation/code/modules/hydroponics/grown/mystery_pod.dm

```diff
@@ -1,4 +1,4 @@
-//Mystery Pod
+//Mystery Pod - Currently not in game, meant to be in replicapod.dm but was removed overtime due to TG edits.
```

Comment:
These are not a tg thing they're added by sushi and are incredibly broken doing things like spawning chem dispensers. We have them here as a thing that could be spawned by admins but they should be essentially unobtainable through normal means.

Reply:
It's editing a TG file to do so, I personally don't think it's worth it, it can be added at a future PR if you think otherwise, or you can try to PR it to TG so we don't have to maintain it.